eks-anywhere-common/Addons/Partner/Pulumi/pulumi-tester-configmap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
If this configmap is for tester, this should be moved to `Testers/Pulumi` folder.